### ISSUE TITLE
Force import of a container fails.

### ISSUE THREAD

**Issue by griznog** [hpcng/warewulf#341]
When I try to force import a container it fails with this message:

```
[root@lepomis ~]# wwctl container import docker://warewulf/rocky rocky8 --force
Overwriting existing VNFS
Updating the container's /etc/resolv.conf
[ERROR]   : Could not create destination file /datapool/warewulf/chroots/rocky8/rootfs/etc/resolv.conf: open /datapool/warewulf/chroots/rocky8/rootfs/etc/resolv.conf: no such file or directory
[WARNING] : Could not copy /etc/resolv.conf into container: open /datapool/warewulf/chroots/rocky8/rootfs/etc/resolv.conf: no such file or directory
Building container: rocky8
[ERROR]   : Could not build container rocky8: Container does not exist
```

`rm -rf` of the chroot directory lets teh import work normally.

**Comment by mslacken** [comment]
The resolv.conf issue is addressed with #665

**Comment by mslacken** [comment]
Also addressed in #474 so closing this as a double
